superbench/runner/playbooks/deploy.yaml

```diff
@@ -100,6 +100,7 @@
           {{ '--gpus=all' if nvidia_gpu_exist else '' }} \
           {{ '--security-opt seccomp=unconfined --group-add video' if amd_gpu_exist else '' }} \
           -w /root -v {{ workspace }}:/root -v /mnt:/mnt \
+          -v /var/run/docker.sock:/var/run/docker.sock -v {{ workspace }}/docker:/var/lib/docker \
```

Comment:
What's the purpose for this?

Reply:
To reuse /var/run/docker.sock on host and mount path on host to /var/lib/docker as the docker storage.
It's a recommended way to use docker inside container.

Reply:
Got it.